elements/footer_copyright.php

```diff
@@ -0,0 +1,5 @@
+<div class="row">
+    <div class="col-md-8 col-xs-12 footer-copyright">
+        <span>Copyright © <?=date('Y')?> BrandCentral. All Rights Reserved.</span>
+    </div>
+</div>
```

Comment:
Not a real issue but make sure this file ends with a single newline like header_logo.php does, this is a PSR-2 requirement